adding variables to preconfigure syncthing and folders

asmodehn opened the pull request:
Just something I am working on at the moment.
Let me know if you want ( more || less ) things in this PR.

Thanks for the feedback !

asmodehn committed:
adding variables to preconfigure syncthing and folders

spk commented:
Hi @asmodehn, thanks for working on that, I don't have so much time currently but I just saw that build is falling for debug [leftover](https://travis-ci.org/spk/ansible-syncthing-debian/builds/141558545#L523).
I will have more time next week.
Cheers

asmodehn committed:
removed debug task

spk commented on tasks/configure.yml:
```diff
@@ -0,0 +1,47 @@
+    - syncthing-configure-directories
+
+# - debug: var=hostvars[inventory_hostname]
+# - debug: var=groups
```
I prefer removing comment; since if you know ansible you already know how to debug it

spk commented on tasks/main.yml:
```diff
@@ -1,43 +1,12 @@
+# tasks file for supervisor
```
Does this comment correspond to the file ?

spk commented:
Build is failing because the default version is old and not present anymore on the syncthing repositories.
A rebase should fix the problem @asmodehn

asmodehn committed:
cleaning up misleading comments

asmodehn committed:
adding default syncthing user to dockerfile

asmodehn committed:
fixed default vars.

asmodehn committed:
updated README

asmodehn commented:
I did the rebase and cleaned up the commits.
Also added the default user to get travis to build correctly.

I also wanted to ask you about the handler. It is using service module, but in my version (ubuntu trusty), syncthing doesn't install itself as a service... So I m using supervisor to manage it (directly in the playbook).

I saw travis also break on that handler... Any idea if & how we could improve that handler to work in all cases ?

asmodehn committed:
using root for install task

asmodehn committed:
commenting handler since it doesnt work.

spk commented:
This project focus on Debian, maybe you want to lookup this other project that work with supervisor https://github.com/escaped/ansible-syncthing

asmodehn committed:
removed become from install tasks

asmodehn commented:
Thanks for the pointer I ll have a look at it...

In the mean time, is there anything I can do to make this PR useful for this repo ?

spk commented:
I did not check locally but the CI saying that the role is not idempotent, cf https://travis-ci.org/spk/ansible-syncthing-debian/builds/144379578#L585
I think also that https://github.com/spk/ansible-syncthing-debian/pull/1/commits/99683427199300d32a25d179256276c41699a99b should be reverted

Commit:
disabling handler notification for now
